sherlock/resources/data.json

```diff
@@ -1240,6 +1240,13 @@
     "username_claimed": "blue",
     "username_unclaimed": "noonewouldeverusethis"
   },
+  "PyPi": {
+  "errorType": "status_code",
+  "url": "https://pypi.org/user/{}",
+  "urlMain": "https://pypi.org",
+  "username_claimed": "Blue",
+  "username_unclaimed": "noonewouldeverusethis"
```

Comment:
Could you indent these lines with two spaces so that it matches the rest of the file?

Reply:
No problem doing that now.